pint/models/spindown.py

```diff
@@ -25,29 +25,28 @@ def __init__(self):
         # The number of terms in the taylor exapansion of spin freq (F0...FN)
         self.num_spin_terms = maxderivs
 
```

Comment:
Fair enough.  (He sounds serious.  You should probably fix this, @luojing1211 ....)